### PR TITLE
Creation of auto command

### PR DESCRIPTION
**Pull request by mastermind-mayhem**

### CODE REVIEW

**Review by End10 — APPROVED** [review]
it look good

**Review by Banana123abcd — COMMENTED** [review]
looks good :)

**Review by awesomeatpiano — APPROVED** [review]

**Review by Julia8585 — APPROVED** [review]

File: src/main/java/frc/robot/commands/Auto/MobilityOneCommand.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.Auto;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+// Hi
```
Comment (mathyourlife) [inline src/main/java/frc/robot/commands/Auto/MobilityOneCommand.java:8]: Looks like this comment was left by mistake.

File: src/main/java/frc/robot/commands/Auto/MobilityOneCommand.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {}
```
Comment (mathyourlife) [inline src/main/java/frc/robot/commands/Auto/MobilityOneCommand.java:22]: Looks like we don't have any code to actually execute yet.  Is that okay?

File: src/main/java/frc/robot/commands/Auto/MobilityOneCommand.java (hunk 1 above)
Comment (mathyourlife) [inline src/main/java/frc/robot/commands/Auto/MobilityOneCommand.java:8]: Looks like this comment was left by mistake.

**Review by mathyourlife — APPROVED** [review]
LGTM

**Review by Banana123abcd — APPROVED** [review]

**Review by beee33 — APPROVED** [review]

**Pull request merged**